templates/patterns/inpage_navigation/pattern-inpage-navigation.html.twig

```diff
@@ -7,11 +7,15 @@
 {% if links is not empty %}
   {{ attach_library('oe_bootstrap_theme/inpage_navigation') }}
 {% endif %}
-{% include '@oe-bcl/inpage-navigation' with {
-  'title': title,
-  'links': links,
-  'content': content,
-  'full_layout': true,
-  'id': inpage_navigation_id,
-} only  %}
-
+<div class="row">
```

Comment:
this is correct here, seems the wrappers where removed from the component itself https://github.com/openeuropa/bootstrap-component-library/blob/f50c7dfeab8695bbb5c18d5f7d5304509512268a/src/compositions/bcl-inpage-navigation/inpage-navigation.story.js

Reply:
Yes but without a content only the sidebar links are displayed. Here (https://oelibrary.netlify.app/default/?path=/story/paragraphs-inpage-navigation--default) in the html tab only the nav links are displayed but in the presentation, the content is also displayed

Reply:
so your question is: should we wrap the hole thing in an if, or what are you saying?